db/seeds.rb

```diff
@@ -6,47 +6,17 @@
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
 
-podcasts = [{uid: "63zeZeyHMgERmbwhDhIrD1", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "4aavXzJwwW4Yy43lAlfjLf", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "5ixqcoDt93GVu6tr6eaiuh", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "0ouPKjo1tKkj8V4nNI7mBv", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "64cCsH4LCyO5U52xUU4Pax", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "2YLFthx5PP55WTazamT0zn", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "31NmrC2kMHlzrORqqOWAF3", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "5phxjbOeUgwqkWfirBFmgJ", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "7hgnfI4D2P5baN9CZxrIw1", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]},
-  {uid: "5kRel8SUkrQyFX25anyPWi", genres: [{"__typename" => "PodcastTopic", "title" => "Documentary", "uri" => "spotify:genre:0JQ5DAqbMKFCWOPjGCVIq4"}]}]
+# db/seeds.rb
 
-# Creating categories
+# Create podcasts
+podcasts = FactoryBot.create_list(:podcast, 40)
```

Comment:
actually this category seed wasn't a 'test' data. I was fetching categories from Spotify manuelly since I can't fetch them via Spotify API. But no worries. I will change this fetching category approach. https://github.com/enderahmetyurt/podiscover/issues/112

Reply:
I overwritten those because validation for podcast:name field was missing which was preventing creation of seed data. Should I be adding them back with valid name values?

Reply:
no worries. but here if someone runs this seed in production, it can overwrite db values :) Maybe we need to add a condition to avoid updating db on production.

Reply:
You can see solution [here](https://github.com/enderahmetyurt/podiscover/pull/137/commits/2f660d9e7ae13f2cb78e4f0e57418b220a8fd1b7)